core/src/main/scala/cromwell/core/package.scala

```diff
@@ -5,9 +5,12 @@ import wdl4s.values.{SymbolHash, WdlValue}
 import scalaz._
 
 package object core {
+  class WorkflowContext(val root: String)
```

Comment:
I'd prefer to not add classes and such to the package object unless there's a good reason to put it in here